### DISCUSSION TITLE
Marine API forecast length

### DISCUSSION THREAD

**Discussion by mexysfr** [open-meteo/open-meteo#336] [Q&A]
Hello here,
I would like to renew my gratitude for the amount of work that has been and is done on this repo !

I was wondering why the marine API forecast length is only 7 days ?
Is there any limitation ?

Thanks again,
Cheers 👍

**Comment by patrick-zippenfenig** [comment]
Right now, there are no models integrated with more than 7 days of wave forecasts. I believe there are additional models available from NOAA with longer forecast, but I didn't check yet
